data/controls.js

```diff
@@ -29,22 +29,78 @@ window.addEventListener('addon-message', function(ev) {
 /*
  * since this file is not bundled, we cannot use the deep-assign
  * package that we are using in the client code.
+ * from sindresorhus/deep-assign
  */
 
-function isObject(item) {
-  return (item && typeof item === 'object' && !Array.isArray(item));
+'use strict';
```

Comment:
Not sure if this works. I thought that `"use strict"` needed to appear before any other statements in a file/function. (per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode)

IIRC, the "use strict" statement will just be ignored though, so I guess no real harm.